src/index.test.js

```diff
@@ -12,16 +13,16 @@ jest.mock('./ccpa', () => ({
 	},
 }));
 
-jest.mock('./tcfv2', () => ({
-	TCFv2: {
-		init: jest.fn(),
-		showPrivacyManager: jest.fn(),
-		willShowPrivacyMessage: () => Promise.resolve('iwillshowit'),
-	},
-}));
+const TCFv2 = {
+	init: jest.spyOn(actualTCFv2, 'init'),
+	showPrivacyManager: jest.spyOn(actualTCFv2, 'showPrivacyManager'),
+	willShowPrivacyMessage: jest.spyOn(actualTCFv2, 'willShowPrivacyMessage'),
+};
```

Comment:
Here’s we can use the `spyOn` method, @sndrs.

Reply:
looks great! should we do the same for CCPA?

Reply:
I’m not sure `willShowPrivacyMessage` is ever called, then. I think this is one for the e2e test?